docs/5_1_scheduled_execution_math.adoc

```diff
@@ -1,15 +1,13 @@
 === Concepts [[concepts-scheduled-execution]]
 
 The Scheduled Execution interface has a different timing concept compared to FMI for Co-Simulation.
-This is required to cover <<clock>> ticks for <<aperiodic-clocks,aperiodic>> <<inputClock,`input clocks`>> which may tick at time a instant that is unpredictable in advance for the simulation algorithm.
+This is required to cover ticks for <<clocks>> of <<causality, `causality == input`>> which may tick at time a instant that is unpredictable in advance for the simulation algorithm.
```

Comment:
I think "Triggered input clocks" are meant here.

Reply:
I tried to improve this sentence (which was a mess even before Matthias tried to improve it, sorry).